coursemapper-kg/concept-map/compose.yaml

```
@@ -1,69 +1,47 @@
+x-common-env: &common-env
+  NEO4J_URI: bolt://neo4j:7687
+  NEO4J_USER: neo4j
+  NEO4J_PASSWORD: password
+  REDIS_HOST: redis
+  REDIS_PORT: 6379
+  REDIS_DB: 0
+  REDIS_PASSWORD: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)